### ISSUE TITLE
I don't see IntelliJ option 

### ISSUE THREAD

**Issue by Yuhtin** [desktop/desktop#12883]
Describe the bug
I not see the IntelliJ Option (i update the github desktop today)

Version & OS
Github Desktop: 2.9.3 (x64)
Intellij: Ultimate 2021.2.1 (212.5080.55)
OS : Windows 10 Pro 21H1

Steps to reproduce the behavior
Go to 'File --> options --> Integrations -->Applications'
Click on ' (External Editor) Drop Down'
Expected behavior
You should be able to see Intellij as an option there.

Actual behavior
You don't see Intellij as an option there

**Comment by sergiou87** [comment]
Thanks for the bug report @Yuhtin!

~~Just curious, which specific version of IntelliJ have you installed?~~ Sorry I just noticed you wrote it in your message 🤦‍♂️ 

Did you see it at some point before GitHub Desktop 2.9.3?

It definitely works for me with the latest version:
![image](https://user-images.githubusercontent.com/1083228/131667323-887af3c4-2a8f-4c7c-9065-20f7139b417b.png)

**Comment by sergiou87** [comment]
@Yuhtin did you install IntelliJ while GitHub Desktop was open? If so, you need to reopen Desktop in order to have it in the list.

If that's not the problem, how did you install it? Was it through Jetbrains Toolbox?

**Comment by callumgarven** [comment]
@sergiou87 Just came to make an issue about this, as I suspected it may be Toolbox for me.

I can't see any of my JetBrain IDEs in Desktop, they're all installed through Jetbrains Toolbox (WebStorm, PHPStorm & Rider)

![image](https://user-images.githubusercontent.com/54223538/131717205-4f8ad51f-6d14-4fa9-b66b-95f72e65cda3.png)


I'm on version 2.9.3 (x64) myself, Windows 10 Pro.

![image](https://user-images.githubusercontent.com/54223538/131717008-55ce4856-bdca-46d3-bda6-4ceb8e489505.png)

I have never saw any JetBrains IDEs in my GitHub Desktop, but I recently switched to installing via Jetbrains Toolbox shortly before I believe you guys recently patched JetBrains IDE detection.

Let me know if I can be of any further help myself.

Looks like this is being tracked here - #9453
